src/applications/vaos/covid-19-vaccine/redux/selectors.js

```diff
@@ -41,7 +41,7 @@ export function getSiteIdForChosenFacility(state) {
 
 export function getChosenSlot(state) {
   const availableSlots = selectCovid19VaccineNewBooking(state).availableSlots;
-  const selectedTime = selectCovid19VaccineFormData(state).date1[0];
+  const selectedTime = selectCovid19VaccineFormData(state)?.date1?.[0];
```

Comment:
Similar question to above for all these `?` additions. Generally, this data should be there, something else might be going on if you were encountering issues with the data not being present.

Reply:
Ah, I added these because some of the selectors were raising errors when a user refreshed the confirmation page, cleared form data, and they weren't being redirected to home. In the case of  any `newBooking` that `?` should be removed since it's always available, and I think it's only needed in `getChosenSlot` since that references a child property of `newAppointment.data.date1`. 

I'll remove the rest now.